{"TimeStamp":"2025-01-21T20:05:30.356108533Z", "Location":"dmesg-cli", "Content":"[    9.984966] raid6: avx2x2   gen() 28322 MB/s\n[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:30.364824334Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/20microsoft on mounted /dev/vda1\nJan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:30.381480075Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:25 attacker systemd[72472]: Reached target Main User Target.\nJan 21 20:05:25 attacker systemd[72472]: Startup finished in 56ms.\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker systemd[1]: session-259.scope: Succeeded.\nJan 21 20:05:28 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1795.\nJan 21 20:05:28 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:28 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[31mERRO#033[0m[20:05:28] [File] error getting file info for: /tmp/sshcmds.12500.sh; stat /tmp/sshcmds.12500.sh: no such file or directory\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:30.392095520Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:30.412556030Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:25 attacker systemd[72472]: Reached target Main User Target.\nJan 21 20:05:25 attacker systemd[72472]: Startup finished in 56ms.\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:25] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:25 attacker systemd[1]: session-259.scope: Succeeded.\nJan 21 20:05:28 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1795.\nJan 21 20:05:28 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:28 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:28] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:28 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[31mERRO#033[0m[20:05:28] [File] error getting file info for: /tmp/sshcmds.12500.sh; stat /tmp/sshcmds.12500.sh: no such file or directory\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:28 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:28] [File] saved file data to core\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:30.421265130Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.052965] raid6: avx2x2   xor() 17956 MB/s\nJan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:35.440569119Z", "Location":"dmesg-cli", "Content":"[    9.984966] raid6: avx2x2   gen() 28322 MB/s\n[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:35.445809922Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/20microsoft on mounted /dev/vda1\nJan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:35.466632542Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] captured dmesg from cli\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] Sending has completed\nJan 21 20:05:31 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1796.\nJan 21 20:05:31 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:31 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:31 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:32 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:32] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker systemd[1]: Started Session 261 of user blankcanvas.\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1797.\nJan 21 20:05:34 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:35.473494007Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sshd[72568]: Accepted publickey for blankcanvas from 192.168.254.2 port 57062 ssh2: RSA SHA256:3I/WEXJMnChkvSdfEeoZ6YTUz2OrccqDruFNY41qR5o\nJan 21 20:05:33 attacker sshd[72568]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 21 20:05:33 attacker systemd-logind[350]: New session 261 of user blankcanvas.\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/mkdir -p /var/log/ttylog/\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.215.trace\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.215.trace\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:35.488775694Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] captured dmesg from cli\nJan 21 20:05:30 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:30] [Logs] Sending has completed\nJan 21 20:05:31 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1796.\nJan 21 20:05:31 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:31 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:31 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:32 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:32] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker systemd[1]: Started Session 261 of user blankcanvas.\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1797.\nJan 21 20:05:34 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:35.494690268Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.052965] raid6: avx2x2   xor() 17956 MB/s\nJan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:40.510449263Z", "Location":"dmesg-cli", "Content":"[    9.984966] raid6: avx2x2   gen() 28322 MB/s\n[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:40.515253234Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/20microsoft on mounted /dev/vda1\nJan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:40.530117229Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:32 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:32] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker systemd[1]: Started Session 261 of user blankcanvas.\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1797.\nJan 21 20:05:34 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] captured dmesg from cli\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] Sending has completed\nJan 21 20:05:37 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1798.\nJan 21 20:05:37 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:37 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:37 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:40 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:40] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:40 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:40] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:40.536657254Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sshd[72568]: Accepted publickey for blankcanvas from 192.168.254.2 port 57062 ssh2: RSA SHA256:3I/WEXJMnChkvSdfEeoZ6YTUz2OrccqDruFNY41qR5o\nJan 21 20:05:33 attacker sshd[72568]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 21 20:05:33 attacker systemd-logind[350]: New session 261 of user blankcanvas.\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/mkdir -p /var/log/ttylog/\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.215.trace\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.215.trace\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:40.559879130Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:32 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:32] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker systemd[1]: Started Session 261 of user blankcanvas.\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:33] [File] saved file data to core\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:33 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:33] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1797.\nJan 21 20:05:34 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:34 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] captured dmesg from cli\nJan 21 20:05:35 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:35] [Logs] Sending has completed\nJan 21 20:05:37 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1798.\nJan 21 20:05:37 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:37 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:37 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:40 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:40] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:40 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:40] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:40.564868042Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.052965] raid6: avx2x2   xor() 17956 MB/s\nJan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:45.580630650Z", "Location":"dmesg-cli", "Content":"[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode\n[ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:45.586968746Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:45.608408086Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:45] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:45 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:45] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:45.615316122Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:05:28 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sshd[72568]: Accepted publickey for blankcanvas from 192.168.254.2 port 57062 ssh2: RSA SHA256:3I/WEXJMnChkvSdfEeoZ6YTUz2OrccqDruFNY41qR5o\nJan 21 20:05:33 attacker sshd[72568]: pam_unix(sshd:session): session opened for user blankcanvas(uid=1001) by (uid=0)\nJan 21 20:05:33 attacker systemd-logind[350]: New session 261 of user blankcanvas.\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/mkdir -p /var/log/ttylog/\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.215.trace\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:33 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.215.trace\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:33 attacker sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:45.632088082Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:45] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:45 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:45] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:45 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:45] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:45.639474960Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:50.656078355Z", "Location":"dmesg-cli", "Content":"[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode\n[ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:50.670544849Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:50.696074530Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:47 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:47] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:47 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:47] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:47 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1801.\nJan 21 20:05:47 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:47 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:47 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:47 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:47] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker systemd[1]: Started Session 262 of user blankcanvas.\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:49 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:49] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:49 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:49] [File] saved file data to core\nJan 21 20:05:49 attacker systemd[1]: session-262.scope: Succeeded.\nJan 21 20:05:49 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:49] [File] saved file data to core\nJan 21 20:05:50 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:50] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:50 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:50] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:50.708151178Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:48 attacker systemd-logind[350]: New session 262 of user blankcanvas.\nJan 21 20:05:48 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/mkdir -p /var/log/ttylog/\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:48 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:48 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:48 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:48 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.216.trace\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:48 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.216.trace\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:48 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:49 attacker sshd[73028]: Received disconnect from 192.168.254.2 port 52362:11: disconnected by user\nJan 21 20:05:49 attacker sshd[73028]: Disconnected from user blankcanvas 192.168.254.2 port 52362\nJan 21 20:05:49 attacker sshd[73022]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 21 20:05:49 attacker systemd-logind[350]: Session 262 logged out. Waiting for processes to exit.\nJan 21 20:05:49 attacker systemd-logind[350]: Removed session 262.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:50.750406778Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:47 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:47] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:47 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:47] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:47 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1801.\nJan 21 20:05:47 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:47 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:47 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:47 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:47] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker systemd[1]: Started Session 262 of user blankcanvas.\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:48] [File] saved file data to core\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:48 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:48] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:49 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:49] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:49 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:49] [File] saved file data to core\nJan 21 20:05:49 attacker systemd[1]: session-262.scope: Succeeded.\nJan 21 20:05:49 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:49] [File] saved file data to core\nJan 21 20:05:50 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:50] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:50 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:50] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:50.764513351Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:55.788580147Z", "Location":"dmesg-cli", "Content":"[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode\n[ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:55.794021270Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:55.811964042Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: session-266.scope: Succeeded.\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: Started Session 267 of user blankcanvas.\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Succeeded.\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Consumed 19.326s CPU time.\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: session-267.scope: Succeeded.\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1803.\nJan 21 20:05:54 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:55.821107594Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.221.trace\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.221.trace\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/kill 72610 72611 72612\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker systemd-logind[350]: Removed session 261.\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sshd[73278]: Received disconnect from 192.168.254.2 port 57364:11: disconnected by user\nJan 21 20:05:53 attacker sshd[73278]: Disconnected from user blankcanvas 192.168.254.2 port 57364\nJan 21 20:05:53 attacker sshd[73272]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 21 20:05:53 attacker systemd-logind[350]: Session 267 logged out. Waiting for processes to exit.\nJan 21 20:05:53 attacker systemd-logind[350]: Removed session 267.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:55.840241279Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: session-266.scope: Succeeded.\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: Started Session 267 of user blankcanvas.\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Succeeded.\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Consumed 19.326s CPU time.\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: session-267.scope: Succeeded.\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1803.\nJan 21 20:05:54 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:05:55.846130043Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:00.859813728Z", "Location":"dmesg-cli", "Content":"[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode\n[ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:00.865230750Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:00.884140784Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Succeeded.\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Consumed 19.326s CPU time.\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: session-267.scope: Succeeded.\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1803.\nJan 21 20:05:54 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] captured dmesg from cli\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] Sending has completed\nJan 21 20:05:57 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1804.\nJan 21 20:05:57 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:57 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:57 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:06:00 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1805.\nJan 21 20:06:00 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:00 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:00 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:06:00 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:00] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:06:00 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:00] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:00.892258492Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/[ -e /tmp/count.blankcanvas ]\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/cat /tmp/count.blankcanvas\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/tee /tmp/count.blankcanvas\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/touch /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.221.trace\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/chmod ugo+rw /var/log/ttylog/ttylog.attacker.infra.real.portscanner.blankcanvas.blankcanvas.221.trace\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sudo: blankcanvas : PWD=/home/blankcanvas ; USER=root ; COMMAND=/usr/bin/kill 72610 72611 72612\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=1001)\nJan 21 20:05:53 attacker systemd-logind[350]: Removed session 261.\nJan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sshd[73278]: Received disconnect from 192.168.254.2 port 57364:11: disconnected by user\nJan 21 20:05:53 attacker sshd[73278]: Disconnected from user blankcanvas 192.168.254.2 port 57364\nJan 21 20:05:53 attacker sshd[73272]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 21 20:05:53 attacker systemd-logind[350]: Session 267 logged out. Waiting for processes to exit.\nJan 21 20:05:53 attacker systemd-logind[350]: Removed session 267.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:00.911499637Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Succeeded.\nJan 21 20:05:53 attacker systemd[1]: session-261.scope: Consumed 19.326s CPU time.\nJan 21 20:05:53 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:05:53] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:05:53 attacker systemd[1]: session-267.scope: Succeeded.\nJan 21 20:05:53 attacker discern-file-sorcerer[2720]: #033[36mINFO#033[0m[20:05:53] [File] saved file data to core\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1803.\nJan 21 20:05:54 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:54 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] captured dmesg from cli\nJan 21 20:05:55 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:05:55] [Logs] Sending has completed\nJan 21 20:05:57 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1804.\nJan 21 20:05:57 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:57 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:05:57 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:06:00 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1805.\nJan 21 20:06:00 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:00 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:00 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:06:00 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:00] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:06:00 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:00] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:00.916964280Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:05.930880856Z", "Location":"dmesg-cli", "Content":"[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode\n[ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:05.937626812Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:05.956599537Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:06:03 attacker systemd[72472]: gpg-agent.socket: Succeeded.\nJan 21 20:06:03 attacker systemd[72472]: Closed GnuPG cryptographic agent and passphrase cache.\nJan 21 20:06:03 attacker systemd[72472]: Removed slice User Application Slice.\nJan 21 20:06:03 attacker systemd[72472]: Reached target Shutdown.\nJan 21 20:06:03 attacker systemd[72472]: systemd-exit.service: Succeeded.\nJan 21 20:06:03 attacker systemd[72472]: Finished Exit the Session.\nJan 21 20:06:03 attacker systemd[72472]: Reached target Exit the Session.\nJan 21 20:06:03 attacker systemd[1]: user@1001.service: Succeeded.\nJan 21 20:06:03 attacker systemd[1]: Stopped User Manager for UID 1001.\nJan 21 20:06:03 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1806.\nJan 21 20:06:03 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:03 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:03 attacker systemd[1]: Stopping User Runtime Directory /run/user/1001...\nJan 21 20:06:03 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:06:03] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:06:03 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:06:03] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:06:03 attacker systemd[1]: run-user-1001.mount: Succeeded.\nJan 21 20:06:03 attacker systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 21 20:06:03 attacker systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 21 20:06:03 attacker systemd[1]: Removed slice User Slice of UID 1001.\nJan 21 20:06:03 attacker systemd[1]: user-1001.slice: Consumed 19.770s CPU time.\nJan 21 20:06:03 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:06:05 attacker discern-jupyter-sorcerer[2749]: #033[36mINFO#033[0m[20:06:05] [Jupyter] File search & saving stopped\nJan 21 20:06:05 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:05] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:06:05 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:05] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:05.966073019Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sshd[73278]: Received disconnect from 192.168.254.2 port 57364:11: disconnected by user\nJan 21 20:05:53 attacker sshd[73278]: Disconnected from user blankcanvas 192.168.254.2 port 57364\nJan 21 20:05:53 attacker sshd[73272]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 21 20:05:53 attacker systemd-logind[350]: Session 267 logged out. Waiting for processes to exit.\nJan 21 20:05:53 attacker systemd-logind[350]: Removed session 267.\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/closerange.bt\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/close.bt\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:05.985010334Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:06:03 attacker systemd[72472]: gpg-agent.socket: Succeeded.\nJan 21 20:06:03 attacker systemd[72472]: Closed GnuPG cryptographic agent and passphrase cache.\nJan 21 20:06:03 attacker systemd[72472]: Removed slice User Application Slice.\nJan 21 20:06:03 attacker systemd[72472]: Reached target Shutdown.\nJan 21 20:06:03 attacker systemd[72472]: systemd-exit.service: Succeeded.\nJan 21 20:06:03 attacker systemd[72472]: Finished Exit the Session.\nJan 21 20:06:03 attacker systemd[72472]: Reached target Exit the Session.\nJan 21 20:06:03 attacker systemd[1]: user@1001.service: Succeeded.\nJan 21 20:06:03 attacker systemd[1]: Stopped User Manager for UID 1001.\nJan 21 20:06:03 attacker systemd[1]: discern-os-record.service: Scheduled restart job, restart counter is at 1806.\nJan 21 20:06:03 attacker systemd[1]: Stopped \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:03 attacker systemd[1]: Started \"The discern network OS metadata recorder. Runs all bpftrace programs\".\nJan 21 20:06:03 attacker systemd[1]: Stopping User Runtime Directory /run/user/1001...\nJan 21 20:06:03 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:06:03] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:06:03 attacker discern-network-sorcerer[2788]: #033[31mERRO#033[0m[20:06:03] [Network] error adding to network slice: error saving network slice: error saving network slice: rpc error: code = Unavailable desc = connection error: desc = \"transport: Error while dialing: dial tcp 10.0.1.1:50051: connect: connection refused\"\nJan 21 20:06:03 attacker systemd[1]: run-user-1001.mount: Succeeded.\nJan 21 20:06:03 attacker systemd[1]: user-runtime-dir@1001.service: Succeeded.\nJan 21 20:06:03 attacker systemd[1]: Stopped User Runtime Directory /run/user/1001.\nJan 21 20:06:03 attacker systemd[1]: Removed slice User Slice of UID 1001.\nJan 21 20:06:03 attacker systemd[1]: user-1001.slice: Consumed 19.770s CPU time.\nJan 21 20:06:03 attacker systemd[1]: discern-os-record.service: Succeeded.\nJan 21 20:06:05 attacker discern-jupyter-sorcerer[2749]: #033[36mINFO#033[0m[20:06:05] [Jupyter] File search & saving stopped\nJan 21 20:06:05 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:05] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:06:05 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:05] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:05.990688087Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:11.008175791Z", "Location":"dmesg-cli", "Content":"[   10.052965] raid6: avx2x2   xor() 17956 MB/s\n[   10.120970] raid6: avx2x1   gen() 23313 MB/s\n[   10.188966] raid6: avx2x1   xor() 14813 MB/s\n[   10.256966] raid6: sse2x4   gen() 13931 MB/s\n[   10.324967] raid6: sse2x4   xor()  7247 MB/s\n[   10.392966] raid6: sse2x2   gen() 15284 MB/s\n[   10.460967] raid6: sse2x2   xor()  9379 MB/s\n[   10.528966] raid6: sse2x1   gen() 12197 MB/s\n[   10.596966] raid6: sse2x1   xor()  7725 MB/s\n[   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\n[   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\n[   10.598665] raid6: using avx2x2 recovery algorithm\n[   10.618897] xor: automatically using best checksumming function   avx       \n[   10.675414] Btrfs loaded, crc32c=crc32c-intel\n[  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\n[  134.833403] Bluetooth: Core ver 2.22\n[  134.834973] NET: Registered protocol family 31\n[  134.835494] Bluetooth: HCI device and connection manager initialized\n[  134.839624] Bluetooth: HCI socket layer initialized\n[  134.841741] Bluetooth: L2CAP socket layer initialized\n[  134.843857] Bluetooth: SCO socket layer initialized\n[ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n[ 5224.537448] device eth1 entered promiscuous mode\n[ 5560.967356] device eth1 left promiscuous mode\n[ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:11.013419773Z", "Location":"/var/log/messages", "Content":"Jan 21 18:24:46 debian 20microsoft: debug: Skipping legacy bootloaders on UEFI system\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/30utility on mounted /dev/vda1\nJan 21 18:24:46 debian 30utility: debug: /dev/vda1 is a FAT32 partition\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/40lsb on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/70hurd on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/80minix on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/83haiku on mounted /dev/vda1\nJan 21 18:24:46 debian 83haiku: debug: /dev/vda1 is not a BeFS partition: exiting\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90linux-distro on mounted /dev/vda1\nJan 21 18:24:46 debian os-prober: debug: running /usr/lib/os-probes/mounted/90solaris on mounted /dev/vda1\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"347\" x-info=\"https://www.rsyslog.com\"] exiting on signal 15.\nJan 21 19:50:53 attacker rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.2102.0]\nJan 21 19:50:53 attacker rsyslogd: [origin software=\"rsyslogd\" swVersion=\"8.2102.0\" x-pid=\"61323\" x-info=\"https://www.rsyslog.com\"] start\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:11.026616324Z", "Location":"/var/log/syslog", "Content":"Jan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-207.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-208.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-209.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-21.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-210.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-211.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-212.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-213.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-22.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-23.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-24.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-25.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-27.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-28.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:10] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:06:11 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:11] [Logs] captured dmesg from cli\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:11.032844890Z", "Location":"/var/log/auth.log", "Content":"Jan 21 20:05:53 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:05:53 attacker sshd[73278]: Received disconnect from 192.168.254.2 port 57364:11: disconnected by user\nJan 21 20:05:53 attacker sshd[73278]: Disconnected from user blankcanvas 192.168.254.2 port 57364\nJan 21 20:05:53 attacker sshd[73272]: pam_unix(sshd:session): session closed for user blankcanvas\nJan 21 20:05:53 attacker systemd-logind[350]: Session 267 logged out. Waiting for processes to exit.\nJan 21 20:05:53 attacker systemd-logind[350]: Removed session 267.\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execve.bt\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/closerange.bt\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/execveat.bt\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/fork.bt\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/kill.bt\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo:     root : PWD=/ ; USER=root ; COMMAND=/usr/bin/bpftrace -f json /etc/discern/bpfsensors/close.bt\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session opened for user root(uid=0) by (uid=0)\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\nJan 21 20:06:03 attacker sudo: pam_unix(sudo:session): session closed for user root\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:11.051546933Z", "Location":"/var/log/daemon.log", "Content":"Jan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-208.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-209.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-21.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-210.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-211.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-212.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-213.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-22.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[36mINFO#033[0m[20:06:10] [Bash] Sent message safely to the Fusion Core\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-23.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-24.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-25.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-27.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:10] [Bash] Error saving attacker-blankcanvas-28.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\nJan 21 20:06:10 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:10] [Logs] /var/log/dmesg not found. trying to capture from cli\nJan 21 20:06:11 attacker discern-logs-sorcerer[2768]: #033[36mINFO#033[0m[20:06:11] [Logs] captured dmesg from cli\nJan 21 20:06:11 attacker discern-bash-sorcerer[72103]: #033[31mERRO#033[0m[20:06:11] [Bash] Error saving attacker-blankcanvas-29.csv core: csv not built. either session hasn't ended or no commands executed. plz don't panic\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
{"TimeStamp":"2025-01-21T20:06:11.061899538Z", "Location":"/var/log/kern.log", "Content":"Jan 21 18:24:46 debian kernel: [   10.120970] raid6: avx2x1   gen() 23313 MB/s\nJan 21 18:24:46 debian kernel: [   10.188966] raid6: avx2x1   xor() 14813 MB/s\nJan 21 18:24:46 debian kernel: [   10.256966] raid6: sse2x4   gen() 13931 MB/s\nJan 21 18:24:46 debian kernel: [   10.324967] raid6: sse2x4   xor()  7247 MB/s\nJan 21 18:24:46 debian kernel: [   10.392966] raid6: sse2x2   gen() 15284 MB/s\nJan 21 18:24:46 debian kernel: [   10.460967] raid6: sse2x2   xor()  9379 MB/s\nJan 21 18:24:46 debian kernel: [   10.528966] raid6: sse2x1   gen() 12197 MB/s\nJan 21 18:24:46 debian kernel: [   10.596966] raid6: sse2x1   xor()  7725 MB/s\nJan 21 18:24:46 debian kernel: [   10.597466] raid6: using algorithm avx2x2 gen() 28322 MB/s\nJan 21 18:24:46 debian kernel: [   10.598087] raid6: .... xor() 17956 MB/s, rmw enabled\nJan 21 18:24:46 debian kernel: [   10.598665] raid6: using avx2x2 recovery algorithm\nJan 21 18:24:46 debian kernel: [   10.618897] xor: automatically using best checksumming function   avx       \nJan 21 18:24:46 debian kernel: [   10.675414] Btrfs loaded, crc32c=crc32c-intel\nJan 21 18:26:50 debian kernel: [  134.731585] alg: No test for fips(ansi_cprng) (fips_ansi_cprng)\nJan 21 18:26:50 debian kernel: [  134.833403] Bluetooth: Core ver 2.22\nJan 21 18:26:50 debian kernel: [  134.834973] NET: Registered protocol family 31\nJan 21 18:26:50 debian kernel: [  134.835494] Bluetooth: HCI device and connection manager initialized\nJan 21 18:26:50 debian kernel: [  134.839624] Bluetooth: HCI socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.841741] Bluetooth: L2CAP socket layer initialized\nJan 21 18:26:50 debian kernel: [  134.843857] Bluetooth: SCO socket layer initialized\nJan 21 19:20:19 debian kernel: [ 3343.637134] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\nJan 21 19:51:40 attacker kernel: [ 5224.537448] device eth1 entered promiscuous mode\nJan 21 19:57:16 attacker kernel: [ 5560.967356] device eth1 left promiscuous mode\nJan 21 20:05:44 attacker kernel: [ 6069.171506] TCP: request_sock_TCP: Possible SYN flooding on port 1339. Sending cookies.  Check SNMP counters.\n", "DevID":"attacker.infra.real.portscanner.blankcanvas"}
